Symbolic link needs additional privileges with Win11.

When I try to configure this repo, I get this error:
```
CMake Error at CMake/Findtinygltf.cmake:80 (file):
  file failed to create symbolic link
  'C:/GitHub/Vulkan-Tutorial/builds/VS2026_64/_deps/tinygltf-src/nlohmann/json.hpp':
  A required privilege is not held by the client.
```

The culprit for this message is line 80 in `attachments/CMake/Findtinygltf.cmake`:
`file(CREATE_LINK "${tinygltf_SOURCE_DIR}/json.hpp" "${tinygltf_SOURCE_DIR}/nlohmann/json.hpp" SYMBOLIC)`

It seems, you need additional privileges to create a symbolic link with Win11.

What is that link used for? When I just skip this line, everthing is configured and generated without issues, and everything builds fine as well.

--- a/attachments/CMake/Findtinygltf.cmake
+++ b/attachments/CMake/Findtinygltf.cmake
@@ -74,12 +74,6 @@ if(NOT tinygltf_INCLUDE_DIR)
                    TINYGLTF_CMAKE_CONTENT "${TINYGLTF_CMAKE_CONTENT}")
     file(WRITE "${tinygltf_SOURCE_DIR}/CMakeLists.txt" "${TINYGLTF_CMAKE_CONTENT}")
 
-    # Create a symbolic link to make nlohmann/json.hpp available
-    if(EXISTS "${tinygltf_SOURCE_DIR}/json.hpp")
-      file(MAKE_DIRECTORY "${tinygltf_SOURCE_DIR}/nlohmann")
-      file(CREATE_LINK "${tinygltf_SOURCE_DIR}/json.hpp" "${tinygltf_SOURCE_DIR}/nlohmann/json.hpp" SYMBOLIC)
-    endif()
-
     # Set tinygltf to header-only mode
     set(TINYGLTF_HEADER_ONLY ON CACHE BOOL "Use header only version" FORCE)
     set(TINYGLTF_INSTALL OFF CACHE BOOL "Do not install tinygltf" FORCE)

